handlers/handlers.go

```diff
@@ -89,14 +111,92 @@ func HandleSessionRequest(
 			panic(err)
 		}
 
+		// Create token as a cookie and set it
+		{
+			userId, err := userservice.GetUserIdFromEmailAddress(loginForm.EmailAddress)
+			if err != nil {
+				panic(err)
+			}
+
+			token, err := createTokenAsString(userId, oneWeekInMinutes)
+			if err != nil {
+				panic(err)
+			}
+
+			expiration := time.Now().Add(oneWeekInMinutes * time.Minute)
```

Comment:
Just so we are clear in the future, Dates without time are `xDate` and dates with time is `xTime` or `xDateTime`

Reply:
agreed that dates w/o time are `xDate`. My instinct for datetimes is to simply say `xTime`. But that's mainly because I've never needed to simply specify a time w/o a date.